model-mesh/dependencies/quickstart.yaml

```
@@ -47,6 +47,35 @@ spec:
           configMap:
             name: etcd-scripts
             defaultMode: 0554
+      initContainers: 
+        - name: etcd-secret-creator
+          image: registry.redhat.io/openshift4/ose-cli@sha256:25fef269ac6e7491cb8340119a9b473acbeb53bc6970ad029fdaae59c3d0ca61
+          command: ["/bin/bash", "-c", "--"]
+          args:
+            - |
+              etcdpasswordexists=$(oc get secrets -o name | grep etcd-passwords || echo "false")
+              modelservingetcdexists=$(oc get secrets -o name | grep model-serving-etcd || echo "false")
+              
+              if [[ $etcdpasswordexists == "false" && $modelservingetcdexists == "false" ]]; then
```

think can have a simple one just check if the variable is set to empty.


```suggestion
               etcdpasswordexists=$(oc get secrets -o name | grep etcd-passwords)
               modelservingetcdexists=$(oc get secrets -o name | grep model-serving-etcd)
               if [ -z $etcdpasswordexists ] && [ -z $modelservingetcdexists ]; then
```